src/type/__tests__/enumType.js

```diff
@@ -59,7 +66,31 @@ describe('Type System: Enum Values', () => {
     }
   });
 
-  var schema = new GraphQLSchema({ query: QueryType });
+  var MutationType = new GraphQLObjectType({
+    name: 'Mutation',
+    fields: {
+      favoriteEnum: {
+        type: FavoriteType,
+        args: { color: { type: ColorType } },
+        resolve(value, color) { return color; }
```

Comment:
That actually did not work when I tried it-- and let me to believe that if a field accepts only one argument (as in this case) then it will not be embedded within an object. There may be a deeper issue here?